lib/mp4/transmuxer.js

```diff
@@ -119,7 +134,66 @@ AudioSegmentStream = function(track) {
     this.trigger('data', {track: track, boxes: boxes});
     this.trigger('done');
   };
+
+  // If the audio segment extends before the earliest allowed dts
+  // value, remove AAC frames until starts at or after the earliest
+  // allowed DTS
```

Comment:
I think we should explain why there is an earliest allowed DTS in the first place. We might have a comment somewhere else but it wouldn't hurt to repeat ourselves a bit in comment form.